src/solver/utils/ortools_utils.cpp

```diff
@@ -116,6 +116,20 @@ static void tuneSolverSpecificOptions(MPSolver* solver)
     }
 }
 
+static bool solverSupportsWarmStart(const MPSolver* solver)
```

Comment:
**Warm** is better than **hot** ? 
I would have no problem with that, but as everybody in the project say **hot start** when she/he talks about that, it can be confusing for a second (is there a hot start and a warn start ? Are they different ? Ok, no, they are the same)

Reply:
I agree that we mostly say "hot start". But it seems that we're wrong since Google returns many more results for "warm start".